<?xml version="1.0"?>
<launch>
    <param name="/use_sim_time" value="true" />

     <node pkg="normal_estimation" type="normal_estimation_node2" name="normal_estimation_node2" output="screen">
        <remap from="lidar_points" to="rslidar_points" />
        <param name="max_row_nn_dist" value="0.07" />
        <param name="max_nn_minor_axis_var" value="0.00005" />
        <param name="min_nn_major_axes_var" value="0.0009" />
        <param name="max_pt_to_normal_ang" value="1.3" />
    </node>
</launch>
